test/README.md

```diff
@@ -1,4 +1,49 @@
 ### How to execute the e2e tests locally
 
-1. Start the vcluster via `devspace run dev` and running `go run -mod vendor cmd/vcluster/main.go start --sync 'networkpolicies'` in the terminal
-2. Then start the e2e tests via `VCLUSTER_SUFFIX=vcluster go test -v -ginkgo.v -ginkgo.skip='.*NetworkPolicy.*'`
+1. Start test environment:
+    - Start the vcluster in test mode `devspace run test <distro> <valuesFilePath from the testsuite>`
+    
+      - To run the test from the a general test suite i.e from `./test/e2e`.
+      
+          `devspace --namespace vcluster run test k3s --debug`
+      
+      - To run tests from a specific test suite, you'll need to specify the values file path from that test suite.
+      
+          `devspace --namespace vcluster run test k3s --var VALUES_FILE=./test/e2e_node/values.yaml --debug`
```

Comment:
nit-pick: the `--debug` flag is probably unnecessary

Reply:
okay